src/plugins/azgenerator/CodeModelAzImpl.ts

```diff
@@ -21,6 +21,7 @@ class ActionParam {
 export class CodeModelCliImpl implements CodeModelAz {
     codeModel: CodeModel;
     options: any;
+    outputFolder: string;
```

Comment:
where is the code to use outputFolder paraeter?

Reply:
removed in latest commit